CareKit/SymptomTracker/OCKSymptomTrackerViewController.m

```diff
@@ -84,8 +84,7 @@ - (void)viewDidLoad {
                                                                               style:UIBarButtonItemStylePlain
                                                                              target:self
                                                                              action:@selector(showToday:)];
-    self.navigationItem.rightBarButtonItem.tintColor = self.progressRingTintColor;
-    
```

Comment:
Yes, I confirm you I'm using the tintColor instead, I managed to do so because in RK I was using that var to get rid of the default color of some elements.

Probably instead using my PR, we can improve the [docs](http://carekit.org/docs/Classes/OCKSymptomTrackerViewController.html#//api/name/progressRingTintColor) adding this note about ```progressRingTintColor```, what do you think?